packages/api/cms-api/src/builds/builds.service.ts

```diff
@@ -70,15 +76,22 @@ export class BuildsService {
         return true;
     }
 
-    async getBuilds(options?: { limit?: number | undefined }): Promise<BuildObject[]> {
+    /**
+     * Should only be used internally
+     */
+    async dangerouslyCreateBuildsWithoutUserCheck(trigger: string): Promise<boolean> {
```

Comment:
could this be named to what it actually does? eg. createBuildsForAllScopes? (If I understand that correctly)

Reply:
I agree. Renamed.